# Genomic Surveillance of SARS-CoV-2 in Puerto Rico
This public repository analyzes SARS-CoV-2 genomes using the [Nextstrain](https://nextstrain.org/) platform to understand how the virus that is responsible for the COVID-19 pandemic, evolves and spreads in Puerto Rico. By reconstructing a phylogeny we can learn about important epidemiological phenomena such as spatial spread, introduction timings and epidemic growth rate.

We maintain (updated weekly) two publicly-available builds, accesible at the [Nextstrain community site](https://nextstrain.org/community/arodzh-sudo/ncov-puertorico/) that incorporates [recent changes](https://github.com/nextstrain/ncov/pull/910) made by the Nextstrain team which splits the phylogeny into a [six-months](https://nextstrain.org/community/arodzh-sudo/ncov-puertorico/Puerto-Rico/six-months/?f_country=Puerto%20Rico&m=div) build that focus subsampling on the past six months and an [all-time](https://nextstrain.org/community/arodzh-sudo/ncov-puertorico/Puerto-Rico/all-time/?f_country=Puerto%20Rico) build that subsamples evenly across time. The original ncov code from the Nextstrain team is available [here](https://github.com/nextstrain/ncov).

The SARS-CoV-2 genomes from Puerto Rico are analyzed against a diverse set of genomes from North America to provide phylogenetic context. Wuhan-Hu-1/2019 is used as a reference for site numbering and genome structure. The phylogeny is rooted relative to early samples of the pandemic and temporal resolution assumes a nucleotide substitution rate of 8 × 10^-4 substitutions per site per year.

Genomic sequences used in these analyses are obtained from [GISAID](https://gisaid.org) and remain subject to GISAID’s [Terms and Conditions](https://www.gisaid.org/registration/terms-of-use/). We gratefully acknowledge the authors, originating and submitting laboratories of the genetic sequences and metadata made available through [GISAID](https://gisaid.org).
