# Nextstrain Builds for Genomic Epidemiology of SARS-CoV-2 in Puerto Rico
SARS-CoV-2 genomes from Puerto Rico analyzed against a diverse set of genomes from the globe to provide phylogenetic context. Site numbering and genome structure uses Wuhan-Hu-1/2019 as reference. The phylogeny is rooted relative to early samples from Wuhan. Temporal resolution assumes a nucleotide substitution rate of 8 Ã— 10^-4 subs per site per year. There were SNPs present in the first and last few bases of the alignments that were masked as likely sequencing artifacts. 

These results are provided as heuristic guides only, based on literature review and phylogenetic analysis. Relationships between variants must be determined in conjunction with clinical, microbiological, and epidemiological information.

I. [Variants of Concern (VOC) and Variants Being Monitored (VBM)](https://nextstrain.org/community/arodzh-sudo/ncov-puertorico/Puerto-Rico/?f_country=Puerto%20Rico)

II. [Delta (B.1.617.2 and AYs)](https://nextstrain.org/community/arodzh-sudo/ncov-puertorico/Puerto-Rico/Delta?f_country=Puerto%20Rico&label=clade:21A%20%28Delta%29)

III. [21K and 21L Omicron](https://nextstrain.org/community/arodzh-sudo/ncov-puertorico/Puerto-Rico/Omicron?f_country=Puerto%20Rico&label=clade:21M%20%28Omicron%29)


